modules/apps/archived/archived-modules-upgrade/src/main/java/com/liferay/archived/modules/upgrade/internal/UpgradeDictionary.java

```diff
@@ -0,0 +1,35 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.archived.modules.upgrade.internal;
+
+import com.liferay.portal.kernel.upgrade.UpgradeProcess;
+
+/**
+ * @author Sam Ziemer
+ */
+public class UpgradeDictionary extends UpgradeProcess {
+
+	@Override
+	protected void doUpgrade() throws Exception {
+		LayoutTypeSettingsUtil.removePortletId(connection, "23");
```

Comment:
@SamZiemer, we should take into account that this portlet can have two different portletIds depending on the original version, 23 for 6.2 or below and com_liferay_dictionary_web_portlet_DictionaryPortlet for higher versions. Please take into account both cases. As a reference, see:
https://github.com/liferay/liferay-portal/blob/master/modules/apps/archived/dictionary-web/src/main/java/com/liferay/dictionary/web/internal/upgrade/DictionaryWebUpgrade.java#L42